### PR TITLE
TOOLS-2721: Add RHEL8 ARM to Tools

### PR DESCRIPTION
**Pull request by huan-Mongo**

### CODE REVIEW

File: release/platform/platform.go [hunk 1]
```diff
@@ -367,4 +367,12 @@ var platforms = []Platform{
 		Repos:     []string{RepoOrg, RepoEnterprise},
 		BuildTags: []string{"ssl", "sasl", "gssapi", "failpoints"},
 	},
+	{
+		Name:      "rhel-82",
+		Arch:      "arm64",
```
Comment (mastertheknife) [inline release/platform/platform.go:372]: I think arch should be "aarch64". In RHEL world its called "aarch64", but in Debian\Ubuntu its indeed "arm64"

Comment (mastertheknife) [reply release/platform/platform.go:372]:
```
root@removed:~ $ rpm --eval '%{_arch}'
aarch64
```

Comment (rychipman) [inline release/platform/platform.go:372]: a couple suggestions:
- let's add a `RPMArch()` method similar to `DebianArch()`
- let's create constants for these architecture strings, similar to what we've done for OS, Pkg, and Repo

**Review by rychipman — CHANGES_REQUESTED** [review]
I think we also need to add `rhel82` entries to `repo-config.yaml`

**Comment by huan-Mongo** [comment]
I have added `rhel82` entries to `repo-config.yaml`, the `RPMArch()` function, as well as the architecture constant.

**Review by rychipman — CHANGES_REQUESTED** [review]

File: release/release.go [hunk 2]
```diff
@@ -214,7 +214,7 @@ func getRPMFileName() string {
 
 	return fmt.Sprintf(
 		"mongodb-database-tools-%s.%s.rpm",
-		vStr, p.Arch,
+		vStr, p.RPMArch(),
```
[the remark this answers is not in the document]
Comment (huan-Mongo) [reply release/release.go:217]: I see. The substitution is added.

File: release/platform/platform.go [hunk 3]
```diff
@@ -140,231 +158,239 @@ var platformsByVariant map[string]Platform
...
-		Arch:      "s390x",
+		Arch:      ArchS390x,
 		OS:        OSLinux,
 		Pkg:       PkgDeb,
 		Repos:     []string{RepoOrg, RepoEnterprise},
 		BuildTags: []string{"ssl", "sasl", "gssapi", "failpoints"},
 	},
+	{
+		Name:      "rhel-82",
+		Arch:      ArchArm64,
```
[the remark this answers is not in the document]
Comment (huan-Mongo) [reply release/platform/platform.go]: Done. This has been regrouped.

File: etc/repo-config.yml [hunk 4]
```diff
@@ -114,6 +114,14 @@ repos:
       - yum/redhat/8/mongodb-org
       - yum/redhat/8Server/mongodb-org
 
+  - name: rhel82
```
[the remark this answers is not in the document]
Comment (huan-Mongo) [reply etc/repo-config.yml:117]: Done!

File: etc/repo-config.yml [hunk 5]
```diff
@@ -320,6 +328,15 @@ repos:
       - yum/redhat/8/mongodb-enterprise
       - yum/redhat/8Server/mongodb-enterprise
 
+
+  - name: rhel82
+    type: rpm
+    edition: enterprise
+    bucket: repo.mongodb.org
```
Comment (rychipman) [inline etc/repo-config.yml:335]:
```suggestion
    bucket: repo.mongodb.com
```

**Review by rychipman — APPROVED** [review]
one typo in the repo config, but otherwise LGTM

**Pull request merged**